### PR TITLE
closes #197 WGs blog

### LINKED ISSUE #197: Blog Post:  pharmaverse council update on WGs and package inclusion
### Blog Post

Short update from recent council discussions that would be good to share with our community via a blog (rather than only a slack post - as these get hidden after 90 days)

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: `c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
- [x] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [x] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
I checked spelling on my side and is fine so seems CI is still broken. Also I wasn't able to render in my session, so please do let me know if anything doesn't look so great in final page such as if the image is too big or anything like that.

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (manciniedoardo) [inline posts/2024-07-24_working__groups_.../working__groups__updates.qmd:31]:
```suggestion
to our full community in future via our [Slack workspace](https://pharmaverse.slack.com/), to make for a more
```

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (manciniedoardo) [inline posts/2024-07-24_working__groups_.../working__groups__updates.qmd:33]:
```suggestion
website – see the FAQ section on the [homepage](https://pharmaverse.org/).
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
LGTM, added two suggestions for extra links

**Review by manciniedoardo — APPROVED** [review]
LGTM

**Comment by rossfarrugia** [comment]
thanks @manciniedoardo - i also shared this PR with the pharmaverse council in case any comments so let's only merge say 1 week from now just to give time for any possible comments. chat at our call tomorrow!

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (StefanThoma) [inline posts/2024-07-24_working__groups_.../working__groups__updates.qmd:37]: Hi @rossfarrugia 
I don't quite understand what the council is now responsible for, if not deciding on which packages go into the pharmaverse. 
Is there a list of responsibilities? 
What do you mean exactly with: 

[...] we plan
to channel any working group type energy towards
[PHUSE DVOST](https://advance.phuse.global/pages/viewpage.action?pageId=327777)
given our strong partnership with PHUSE.

Does this mean that any ro most future work done by the council will be done to support DVOST or in consultation with DVOST?

**Review by StefanThoma — CHANGES_REQUESTED** [review]
Please specify if possible the current responsibilities of the council

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (StefanThoma) [inline posts/2024-07-24_working__groups_.../working__groups__updates.qmd:32]: Does this mean that this is now a democratic process, i.e. with voting on the packages or something? Or how exactly will the community decide to include the packages under the pharmaverse?

**Review by bms63 — COMMENTED** [review]
I think I am bit perplexed on this as well.  

Is it really direct democracy or quorum-based, i.e. if 500 members say approve is it included, or something else?  

Does the package maintainer have to give a case on why they should be included?

Who gets to vote?  Can anyone who joins the slack channel get a chance to vote or is there a threshold to be able to vote.

Anyways lots of questions.  I'm not really passionate about any of this, but just found it unclear on the process.  Happy for what every happens.  Momentum is always nice.

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (rossfarrugia) [reply posts/2024-07-24_working__groups_.../working__groups__updates.qmd:37]: @StefanThoma the council always delegated the responsibility of package inclusion to the WGs which we are now disbanding in favour of letting the community decide. 

Council responsibilities are explained on a separate page: https://pharmaverse.org/contribute/council/

The council is fully committed to continue to grow and strengthen pharmaverse independent of any PHUSE support. The blog is therefore not changing anything in the council - it's about our WGs where we see more potential by utilising PHUSE DVOST and channeling energy that way instead of starting our own pharmaverse WGs to solve specific industry OS challenges. This was already happening really - as although we gave our WGs freedom to decide their own scope they all only focused on pharmaverse package inclusion, as any other wider OS topics already have industry WGs that could be joined and supported.

I'll add a note to the blog to reiterate that none of this impacts council.

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (rossfarrugia) [reply posts/2024-07-24_working__groups_.../working__groups__updates.qmd:32]: I didn't want to make it overly prescriptive and rules-based as we should be able to evolve this as we learn from how it goes, but general thinking:

- we share a new package request to pharmaverse slack and ask community to either thumbs up on GH issue, or add a comment if any concerns
- if any concerns tag the requestor so they get a chance to respond and potentially allay the concern
- if any comment is not resolved and even just 1 member of our community still has an issue with a package being included then we'll take it to the council to adjudicate and a transparent decision with rationale will be added to the GH issue

So everyone has a voice... even 1 single member could stop a package being included if they have a good enough justification! If we don't hear any concerns (and the package fits scope of pharmaverse and inclusion criteria) then we'll go ahead and add it. 

If at any time in future a member of the community raises a concern then we could re-visit and see if any package should be removed.

I'll add a little bit more to the blog but don't want to go over the top either with explaining this.

FYI @bms63

**Comment by rossfarrugia** [comment]
@StefanThoma any chance we could get this re-reviewed and merged early this week? would be good to post this blog out to the community soon given the first such open community package review has already started now last week. note that the link check fail is from an older blog.

**Comment by StefanThoma** [comment]
> @StefanThoma any chance we could get this re-reviewed and merged early this week? would be good to post this blog out to the community soon given the first such open community package review has already started now last week. note that the link check fail is from an older blog. Also spelling is fine on my side so not sure if the CI check is still broken here.

Yeah no problem, I can do it today.

File: inst/WORDLIST.txt
Comment (StefanThoma) [inline inst/WORDLIST.txt:846]:
```suggestion
requestor
pageId
viewpage
```

**Comment by rossfarrugia** [comment]
thanks @StefanThoma - weird, i ran the CICD.R line by line for the spelling updates. maybe i don't have latest spelling package installed 😕

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (StefanThoma) [reply posts/2024-07-24_working__groups_.../working__groups__updates.qmd]: Would you mind adding this to the blog as well? 
Or maybe hint towards this, if it is not quite clear yet?

**Comment by manciniedoardo** [comment]
Hi @rossfarrugia, we discussed this post in the pharmaverseblog meeting today - we feel like the enhancements requested by @StefanThoma would improve the post and make it more referenceable in the future. We'd be happy to merge after they are added.

One idea is to make a flow chart of the decision-making process - what do you think?

File: posts/2024-07-24_working__groups_.../working__groups__updates.qmd
Comment (rossfarrugia) [reply posts/2024-07-24_working__groups_.../working__groups__updates.qmd]: @StefanThoma which bit exactly?

sorry, i added the following as a more concise summary of the above but happy to add some extra lines for whichever bits missed which you feel i should add

```
We don't want to be overly prescriptive with how this process will work as it
might evolve over time, but we'll be openly sharing any future package
applications to pharmaverse via our Slack and we welcome your input to help make
the call on which to accept/decline. Any individual from our community could
express support via a thumbs up on the GitHub issue or add a comment with any
concerns. If the requestor is unable to resolve any concerns, then ultimately
any contentious applications will be raised to the pharmaverse council to
adjudicate. The final decision and rationale will then always be documented on
the issue.
```

**Comment by rossfarrugia** [comment]
@manciniedoardo i made an extra commit for Stefan's comments but happy to add more as needed dependent on the reply above. i think a flowchart would be overkill personally and my main thinking here is that we'll probably see how this goes and evolve it over time and i wouldn't want to keep coming back to update finer details here.

**Review by StefanThoma — APPROVED** [review]
LGTM, thanks @rossfarrugia for the quick updates!

**Comment by manciniedoardo** [comment]
@rossfarrugia thanks, now merged - do you want to publicise given it's on behalf of the council?

**Pull request merged**
